yb-voyager/cmd/exportSchema.go

```diff
@@ -378,12 +380,33 @@ func applyShardingRecommendationIfMatching(sqlInfo *sqlInfo, shardedTables []str
 		return formattedStmt, false, nil
 	}
 	createTableStmt := createStmtNode.CreateStmt
-
-	// Extract schema and table name
 	relation := createTableStmt.Relation
-	parsedTableName := relation.Schemaname + "." + relation.Relname
-	if !slices.Contains(shardedTables, parsedTableName) {
+
+	// true -> oracle, false -> PG
+	parsedTableName := lo.Ternary(relation.Schemaname == "", relation.Relname,
+		relation.Schemaname+"."+relation.Relname)
+
+	match := false
+	switch source.DBType {
+	case POSTGRESQL:
+		match = slices.Contains(shardedTables, parsedTableName)
+	case ORACLE:
+		// TODO: handle case-sensitivity properly
+		for _, shardedTable := range shardedTables {
+			parts := strings.Split(shardedTable, ".")
```

Comment:
How will reserved words table names are stored in the `shardedTable`?

Reply:
shardedTable is the information we fetch from the source db first.
There we fetch from system catalog tables. In which the names are stored in a case sensitive way but without any quoting.

For example:
1. Normal table as per oracle format will be stored as: `EMPLOYEES`
2. Case sensitive table will be stored as this: `EmployEEs`
3. Reserved words table will be stored as case 2

Reply:
Okay, so just to confirm pg parser always gives unquoted name right?

Reply:
yes, i tested the code with case-sensitivity-reserved-words in oracle and it passed.